### PR TITLE
Data: BDFL669.json

### PR DESCRIPTION
**Pull request by BDFL669**
Updating my LinkFree profile data

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7666"></a>

### CODE REVIEW

File: data/BDFL669.json [hunk 1]
```diff
@@ -1,21 +1,69 @@
...
+      "date": "Feb 2022",
+      "icon": "SiHashnode",
+      "color": "Blue",
+      "description": "Started Blogging on Dev.to",
+      "url": "https://dev.to/anvilicious"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/botyliciouss",
+      "icon": "FaTwitter"
```
Comment (MahirMahdi) [inline data/BDFL669.json:60]: Please add a `,` here.

File: data/BDFL669.json [hunk 2]
```diff
@@ -1,21 +1,69 @@
...
+      "title": "Started Open Source Contribution",
+      "date": "June 2022",
+      "icon": "FaGithub",
```
Comment (MahirMahdi) [inline data/BDFL669.json:63]: Please remove this icon as there's already a Twitter icon.

File: data/BDFL669.json [hunk 3]
```diff
@@ -1,21 +1,69 @@
...
+      "color": "red",
+      "description": "Learning while contributing | Contributions List",
+      "url": "https://github.com/BDFL669"
```
Comment (MahirMahdi) [inline data/BDFL669.json:66]: Please remove this `url` there's already a `url` for twitter.

File: data/BDFL669.json [hunk 4]
```diff
@@ -1,21 +1,69 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/botyliciouss",
+      "icon": "FaTwitter"
+      "title": "Started Open Source Contribution",
+      "date": "June 2022",
+      "icon": "FaGithub",
+      "color": "red",
+      "description": "Learning while contributing | Contributions List",
```
Comment (MahirMahdi) [inline data/BDFL669.json:65]: After removing the github url, make sure to remove `,` from here

File: data/BDFL669.json [hunk 5]
```diff
@@ -1,21 +1,69 @@
...
+      "url": "https://www.linkedin.com/in/nigel-okoth-104609206/"
+    }
+  ],
+  "testimonials": [
+
+  ],
   "links": [
     {
-      "name": "Follow me on Twitter",
+      "name": "Follow me on GitHub",
```
Comment (MahirMahdi) [inline data/BDFL669.json:28]: Please remove this name. You've already provided a name for this link object.

**Review by MahirMahdi — CHANGES_REQUESTED** [review]
After making the requested changes, please make sure to validate it here https://linkfree.io/playground

File: data/BDFL669.json (hunk 1 above)
Comment (BDFL669) [reply data/BDFL669.json]: Thank you for your feedback. I have made the necessary changes and validated them on the playground

**Comment by BDFL669** [comment]
I have installed the LinkFree repo on my local environment but it fails to
load my profile even though I am currently able to view it on the official
linkfree.io site.

**Review by MahirMahdi — APPROVED** [review]

**Comment by eddiejaoude** [comment]
> I have installed the LinkFree repo on my local environment but it fails to load my profile even though I am currently able to view it on the official linkfree.io site.
> […](#)

You need to visit a url to load the data, you can read more about it in the docs at the top of the `local development` page https://linkfree.io/docs/environments/local-development

**Comment by SaraJaoude** [comment]
I can see that you have opened a new Pull Request with identical information: https://github.com/EddieHubCommunity/LinkFree/pull/7673/files so we will take it that the last Pull Request is the one you want. 

I will close this Pull Request - in future, when you no longer require Pull Request, please close this promptly yourself, to assist maintainers with managing the project and reduce confusion.

**Pull request closed without merging**